ddb-dm-screen.user.js

```diff
@@ -7,246 +7,201 @@
-                        $('#character-details-'+character.name.replace(/[^0-9a-zA-Z]+/g, '')+' > tbody:last-child').append('<tr><td>Intelligence</td><td>'+ character.stats.int.value + '</td><td>' + character.stats.int.bonus() + '</td><td>' + character.stats.int.savingThrow(character.proficiency) + '</td></tr>');
-                        $('#character-details-'+character.name.replace(/[^0-9a-zA-Z]+/g, '')+' > tbody:last-child').append('<tr><td>Wisdom</td><td>'+ character.stats.wis.value + '</td><td>' + character.stats.wis.bonus() + '</td><td>' + character.stats.wis.savingThrow(character.proficiency) + '</td></tr>');
-                        $('#character-details-'+character.name.replace(/[^0-9a-zA-Z]+/g, '')+' > tbody:last-child').append('<tr><td>Charisma</td><td>'+ character.stats.cha.value + '</td><td>' + character.stats.cha.bonus() + '</td><td>' + character.stats.cha.savingThrow(character.proficiency) + '</td></tr>');
-                        $('#character-details-'+character.name.replace(/[^0-9a-zA-Z]+/g, '')+' > tbody:last-child').append('<tr><td>Proficiency</td><td>+'+ character.proficiency + '</td><td></td><td></td></tr>');
-                        $('#character-details-'+character.name.replace(/[^0-9a-zA-Z]+/g, '')+' > tbody:last-child').append('<tr><td>HP</td><td>'+ character.currentHP + '/' + character.maxHP + '</td><td></td><td></td></tr>');
-                        $('#character-details-'+character.name.replace(/[^0-9a-zA-Z]+/g, '')+' > tbody:last-child').append('<tr><td>Passive Perception</td><td>'+ character.skills.perception.getPassive(character.stats) + '</td><td></td><td></td></tr>');
-                        $('#character-details-'+character.name.replace(/[^0-9a-zA-Z]+/g, '')+' > tbody:last-child').append('<tr><td>Passive Investigation</td><td>'+ character.skills.investigation.getPassive(character.stats) + '</td><td></td><td></td></tr>');
-                        $('#character-details-'+character.name.replace(/[^0-9a-zA-Z]+/g, '')+' > tbody:last-child').append('<tr><td>Passive Insight</td><td>'+ character.skills.insight.getPassive(character.stats) + '</td><td></td><td></td></tr>');
-
-                    }
-                 };
-                xmlhttp.open("GET", charString, true);
-                xmlhttp.send();
-
-            };
+  $('#site').after('<div id="iframeDiv" style="display: none;"></div>');
+  $('.ddb-campaigns-character-card-footer-links-item-view').each(function(index, value) {
+      let node = $(this);
+
+      if (!node.parents().hasClass('ddb-campaigns-detail-body-listing-inactive')) {
+        var request = new XMLHttpRequest();
+        request.onreadystatechange = function() {
+          if (this.readyState == 4 && this.status == 200) {
+            let json = JSON.parse(this.responseText).character;
+            let character = new Character(json);
+
+            $('#iframeDiv').append(`<iframe id="frame-${character.id}"  src="${node.attr('href')}"></iframe>`);
```

Comment:
Lets add `width="1000"` to this to trigger tablet mode where stats and skills are on the same page.

Reply:
Good idea, I'm working on another commit that will add everything we talked about in this pull request.

Reply:
adding width="1000" doesn't seem to trigger tablet mode on my machine at least or the selector for skills is wrong. Any idea?

Reply:
It seems that you need to toggle the iframe visibility for a second or so for it to register it's dimmensions.